### PR TITLE
reference: update sql admin syntax

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? 

Update `admin` syntax for TiDB 4.0.

### What is the related PR or file link(s)? 

- [ ] Reference link(s):
- [x] This PR is to align with: [pingcap/docs-cn#2204](https://github.com/pingcap/docs-cn/pull/2204)
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [ ] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

File: dev/reference/sql/statements/admin.md [hunk 1]
```diff
@@ -8,16 +8,48 @@ category: reference
 
 This statement is a TiDB extension syntax, used to view the status of TiDB and check the data of tables in TiDB.
 
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL;
+```
+
+`ADMIN SHOW DDL`: To view the currently running DDL jobs.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]: How about moving this explanation forward?

File: dev/reference/sql/statements/admin.md [hunk 2]
```diff
@@ -8,16 +8,48 @@ category: reference
 
 This statement is a TiDB extension syntax, used to view the status of TiDB and check the data of tables in TiDB.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion

To view the currently running DDL jobs, use `ADMIN SHOW DDL`:

```

File: dev/reference/sql/statements/admin.md [hunk 3]
```diff
@@ -8,16 +8,48 @@ category: reference
...
 
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL;
+```
+
+`ADMIN SHOW DDL`: To view the currently running DDL jobs.
+
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
To view all the results in the current DDL job queue (including tasks that are running and waiting to be run) and the last ten results in the completed DDL job queue, use `ADMIN SHOW DDL JOBS`:
```

File: dev/reference/sql/statements/admin.md [hunk 4]
```diff
@@ -8,16 +8,48 @@ category: reference
...
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL JOBS [NUM] [WHERE where_condition];
+```
+
+* `NUM`: To view the last `NUM` results in the completed DDL job queue. If not specified, `NUM` is by default 10.
+* `WHERE`: To add filter conditions.
+
+`ADMIN SHOW DDL JOBS`: To view all the results in the current DDL job queue (including tasks that are running and waiting to be run) and the last ten results in the completed DDL job queue.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]: ditto

File: dev/reference/sql/statements/admin.md [hunk 5]
```diff
@@ -27,23 +59,68 @@ ADMIN CHECK TABLE tbl_name [, tbl_name] ...
 
 ## Examples
 
+Run the command below to view the last 10 completed DDL jobs in the currently running DDL job queue. When `NUM` is not specified, only the last 10 completed DDL jobs is presented by default.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
Run the following command to view the last 10 completed DDL jobs in the currently running DDL job queue. When `NUM` is not specified, only the last 10 completed DDL jobs is presented by default.
```

File: dev/reference/sql/statements/admin.md [hunk 6]
```diff
@@ -27,23 +59,68 @@ ADMIN CHECK TABLE tbl_name [, tbl_name] ...
...
+| 38     | test    | t1         | create table  | public               | 32        | 37       | 0         | 2019-01-10 12:32:41.956 +0800 CST | 2019-01-10 12:32:43.956 +0800 CST | synced        |
+| 36     | test    |            | drop table    | none                 | 32        | 34       | 0         | 2019-01-10 11:29:59.982 +0800 CST | 2019-01-10 11:30:00.45 +0800  CST | synced        |
+| 35     | test    |            | create table  | public               | 32        | 34       | 0         | 2019-01-10 11:29:40.741 +0800 CST | 2019-01-10 11:29:41.682 +0800 CST | synced        |
+| 33     | test    |            | create schema | public               | 32        | 0        | 0         | 2019-01-10 11:29:22.813 +0800 CST | 2019-01-10 11:29:23.954 +0800 CST | synced        |
++--------+---------+------------+---------------------+----------------+-----------+----------+-----------+-----------------------------------+-----------------------------------+---------------+
+```
+
+Run the command below to view the last 5 completed DDL jobs in the currently running DDL job queue:
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
Run the following command to view the last 5 completed DDL jobs in the currently running DDL job queue:
```

File: dev/reference/sql/statements/admin.md [hunk 7]
```diff
@@ -27,23 +59,68 @@ ADMIN CHECK TABLE tbl_name [, tbl_name] ...
...
+| 43     | test    | t1         | add index     | public               | 32        | 37       | 6         | 2019-01-10 12:35:13.66 +0800 CST  | 2019-01-10 12:35:14.925 +0800 CST | synced        |
+| 42     | test    | t1         | drop index    | none                 | 32        | 37       | 0         | 2019-01-10 12:34:35.204 +0800 CST | 2019-01-10 12:34:36.958 +0800 CST | synced        |
+| 41     | test    | t1         | add index     | public               | 32        | 37       | 0         | 2019-01-10 12:33:22.62 +0800 CST  | 2019-01-10 12:33:24.625 +0800 CST | synced        |
+| 40     | test    | t1         | drop column   | none                 | 32        | 37       | 0         | 2019-01-10 12:33:08.212 +0800 CST | 2019-01-10 12:33:09.78 +0800 CST  | synced        |
++--------+---------+------------+---------------------+----------------+-----------+----------+-----------+-----------------------------------+-----------------------------------+---------------+
+```
+
+Run the command below to view the uncompleted DDL jobs in the test database. The results include the DDL jobs that are running and the last 5 DDL jobs that are completed but failed.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
Run the following command to view the uncompleted DDL jobs in the test database. The results include the DDL jobs that are running and the last 5 DDL jobs that are completed but failed.
```

**Comment by ran-huang** [comment]
Committed all the suggestions. PTAL~ @lilin90

File: dev/reference/sql/statements/admin.md [hunk 8]
```diff
@@ -8,16 +8,47 @@ category: reference
...
+To view the currently running DDL jobs, use `ADMIN SHOW DDL`:
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL;
+```
+
+To view all the results in the current DDL job queue (including tasks that are running and waiting to be run) and the last ten results in the completed DDL job queue, use `ADMIN SHOW DDL JOBS`:
+{{< copyable "sql" >}}
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion

{{< copyable "sql" >}}
```

File: dev/reference/sql/statements/admin.md [hunk 9]
```diff
@@ -8,16 +8,47 @@ category: reference
...
+
+```sql
+ADMIN SHOW DDL JOBS [NUM] [WHERE where_condition];
+```
+
+* `NUM`: To view the last `NUM` results in the completed DDL job queue. If not specified, `NUM` is by default 10.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]: Generally, we use a lowercase letter to begin the text immediately following a colon.

```suggestion
* `NUM`: to view the last `NUM` results in the completed DDL job queue. If not specified, `NUM` is by default 10.
```

File: dev/reference/sql/statements/admin.md [hunk 10]
```diff
@@ -8,16 +8,47 @@ category: reference
...
+* `WHERE`: To add filter conditions.
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
* `WHERE`: to add filter conditions.
```

**Comment by ran-huang** [comment]
@lilin90 PTAL~

**Pull request merged**